src/Validator/Constraints/EventCanStartSlotValidator.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Validator\Constraints;
+
+use App\Dto\SlotRequest;
+use Symfony\Component\Form\Form;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+final class EventCanStartSlotValidator extends ConstraintValidator
+{
+    public function validate($value, Constraint $constraint): void
```

Comment:
missing value type

Reply:
Can't add value type here because it's an inherited function of ConstraintValidatorInterface

Reply:
Yeap didn't see that :P